scipy/optimize/_slsqp_py.py

```diff
@@ -250,6 +250,45 @@ def _minimize_slsqp(func, x0, args=(), jac=None, bounds=None,
 
         .. versionadded:: 1.16.0
 
+    Returns
+    -------
+    res : OptimizeResult
+        The optimization result represented as an ``OptimizeResult`` object.
+        In this dict-like object the following fields are of particular importance:
+        ``x`` the solution array, ``success`` a Boolean flag indicating if the
+        optimizer exited successfully, ``message`` which describes the reason for
+        termination, and ``multipliers`` which contains the Karush-Kuhn-Tucker
+        (KKT) multipliers for the QP approximation used in solving the original
+        nonlinear problem. See ``Notes`` below. See also `OptimizeResult` for a
+        description of other attributes.
+
+    Notes
+    -----
+    The KKT multipliers are returned in the ``OptimizeResult.multipliers``
+    attribute as a NumPy array. Denoting the dimension of the equality constraints
+    with ``meq``, and of inequality constraints with ``mineq``, then the returned
```

Comment:
nit ``mineq`` is never referenced, do we need to introduce it?

Reply:
Insufficient coffee 🤦‍♂️